src/main.ts

```diff
@@ -47,6 +52,7 @@ export const config = {
     // Filters are unions
     events: [
       {
+        fromAddress: KAKAROT_ADDRESS,
```

Comment:
does this work? because Kakarot doesn't emit `TRANSACTION_EXECUTED`, not sure this will work

Reply:
I think you need to filter on `TRANSACTION_EXECUTED` and then refilter these on `KAKAROT_ADDRESS`. Or maybe filter in the `transform` function the transactions with `destination` == KAKAROT_ADDRESS.